docs/antctl.md

```diff
@@ -14,18 +14,20 @@ running in two different modes:
 ## Table of Contents
 
 <!-- toc -->
-- [Installation](#installation)
-- [Usage](#usage)
-  - [Showing or changing log verbosity level](#showing-or-changing-log-verbosity-level)
-  - [Collecting support information](#collecting-support-information)
-  - [controllerinfo and agentinfo commands](#controllerinfo-and-agentinfo-commands)
-  - [NetworkPolicy commands](#networkpolicy-commands)
-    - [Mapping endpoints to NetworkPolicies](#mapping-endpoints-to-networkpolicies)
-  - [Dumping Pod network interface information](#dumping-pod-network-interface-information)
-  - [Dumping OVS flows](#dumping-ovs-flows)
-  - [OVS packet tracing](#ovs-packet-tracing)
-  - [Traceflow](#traceflow)
-  - [Antctl Proxy](#antctl-proxy)
+- [Antctl](#antctl)
```

Comment:
No need to change here.

Reply:
sure, I guess it's auto-formated by vscode, let me revert this.